appyters/Reactome_Harmonizome_ETL/Reactome.ipynb

```diff
@@ -483,21 +691,121 @@
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xlabel = 'UMAP 1'\n",
+    "ylabel = 'UMAP 2'\n",
+    "\n",
+    "source2 = ColumnDataSource(\n",
+    "        data=dict(\n",
+    "            x = mapped_df.x,\n",
+    "            y = mapped_df.y,\n",
+    "            alpha = [0.7] * mapped_df.shape[0],\n",
+    "            colors = mapped_df['color'], \n",
+    "            size = [6] * mapped_df.shape[0],\n",
+    "            gene_set = mapped_df['term'],\n",
+    "            cluster = mapped_df['cluster']\n",
+    "        )\n",
+    "    )\n",
+    "\n",
+    "hover_emb = HoverTool(names=[\"df\"], tooltips=\"\"\"\n",
+    "    <div style=\"margin: 10\">\n",
+    "        <div style=\"margin: 0 auto; width:300px;\">\n",
+    "            <span style=\"font-size: 12px; font-weight: bold;\">Gene Set:</span>\n",
+    "            <span style=\"font-size: 12px\">@gene_set</span>\n",
+    "        <div style=\"margin: 0 auto; width:300px;\">\n",
+    "            <span style=\"font-size: 12px; font-weight: bold;\">Coordinates:</span>\n",
+    "            <span style=\"font-size: 12px\">(@x,@y)</span>\n",
+    "        <div style=\"margin: 0 auto; width:300px;\">\n",
+    "            <span style=\"font-size: 12px; font-weight: bold;\">Cluster:</span>\n",
+    "            <span style=\"font-size: 12px\">@cluster</span>\n",
+    "        </div>\n",
+    "    </div>\n",
+    "    \"\"\")\n",
+    "\n",
+    "tools_emb = [hover_emb, 'pan', 'wheel_zoom', 'reset', 'save']\n",
+    "title_emb = 'Gene Sets in Reactome Library'\n",
+    "plot_emb = figure(plot_width=1000, plot_height=700, tools=tools_emb, title=title_emb, x_axis_label=xlabel, y_axis_label=ylabel)\n",
+    "plot_emb.circle( 'x', 'y', source = source2, size='size',\n",
+    "                alpha='alpha', line_alpha=0, line_width=0.01, name=\"df\", \n",
+    "                fill_color = 'colors', \n",
+    "                line_color=\"black\", hover_fill_color=\"firebrick\")\n",
+    "plot_emb.xaxis.axis_label_text_font_style = 'normal'\n",
+    "plot_emb.xaxis.axis_label_text_font_size = '18px'\n",
+    "plot_emb.yaxis.axis_label_text_font_size = '18px'\n",
+    "plot_emb.yaxis.axis_label_text_font_style = 'normal'\n",
+    "plot_emb.title.align = 'center'\n",
+    "plot_emb.title.text_font_size = '18px'\n",
+    "\n",
+    "show(plot_emb)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Link to download output files: [click here](./output_archive.zip)"
+    "# Downloadable File Links"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "uf.archive(path, output_name)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "display(FileLink('reactome_output_archive.zip', result_html_prefix='Archive of all files: '))\n",
```

Comment:
Given the way uf.archive is implemented, the file will actually be `reactomeoutput_archive.zip`